docusaurus/docusaurus.config.js

```diff
@@ -84,6 +79,12 @@ module.exports = {
   projectName: 'stream-chat',
   tagline: 'Stream Chat official component SDKs',
   themeConfig: {
+    algolia: {
+      apiKey: 'fd1b03de28081b5aa29dbccced4620b9',
```

Comment:
This is a read-only api key. I think it doesnt make sense to add it to environment variables right now once we will end up exposing it on the gzip file anyways.